### ISSUE TITLE
Completion for dynamic attributes and properties 

### ISSUE THREAD

**Issue by vanarok** [hawkticehurst/webc-vscode#2]
I noticed that completion isn't offered for input in [dynamic properties and attributes](https://www.11ty.dev/docs/languages/webc/#dynamic-attributes-and-properties).

**Comment by hawkticehurst** [comment]
Yep, that's correct. I never got around to implementing completions for WebC's more custom syntax in the initial round of implementing all of this.

As I just mentioned in #1, I'm still uncertain that I want to continue building/maintaining this project since I'm tight on free time, but if you would like to contribute a PR with this change or fork this repo to make your own version of the project you have my full permission to do so!
